src/components/ChannelPreview/ChannelPreviewMessenger.tsx

```diff
@@ -70,6 +71,7 @@ const UnMemoizedChannelPreviewMessenger = <
               {unread}
             </div>
           )}
+          {messageDeliveryStatus && <span>{messageDeliveryStatus}</span>}
```

Comment:
Do we intentionally want to write `read` or `delivered` in plain text here?

Reply:
Will remove this one. This PR should not introduce any change to `ChannelPreviewMessenger`.

Reply:
Done